spyder/app/tour.py

```diff
@@ -584,9 +582,17 @@ def toolbutton(icon):
 
         arrow = get_image_path('hide.png')
 
+        if self.parent.styleSheet():
+            self.color_top = QColor.fromRgb(35, 38, 41)
+            self.color_back = QColor.fromRgb(35, 38, 41)
+            light_background = ''
+        else:
+            self.color_top = QColor.fromRgb(230, 230, 230)
+            self.color_back = QColor.fromRgb(255, 255, 255)
+            light_background = 'background-color: rgbs(230,230,230,100%);'

```

Comment:
I think these colors shouldn't be hardcoded. Can't we define them as constants somewhere, as we did with `MAIN_FG_COLOR`?